### PR TITLE
Update game_database.json

### PR DESCRIPTION
**Pull request by ThePastorJ**
Added some of my favourite games to the list, and organised the list alphabetically.

### CODE REVIEW

**Comment by adirh3** [comment]
@ThePastorJ thanks a lot, you just made me realize I made a mistake with the schema.
It does not support multiple games EXE per profiles.
I made change for that - https://github.com/adirh3/Sonar.AutoSwitch/pull/17

Please update your PR accordingly, thanks!

**Comment by ThePastorJ** [comment]
Just updated. I'm making the assumption that any rules made in the app will override the database rules? If not, that'd be a change I'd suggest so people can make their own tweaks on what I've put here.

File: game_database.json [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
+	"ExeName": "bg3_dx11"
   },
   {
-    "SonarProfileName": "Cyberpunk 2077",
-    "ExeName": "Cyberpunk2077"
-  },  
+	"SonarProfileName": "Cyberpunk 2077",
+  "ExeName": "Cyberpunk2077"
+  },
+  {
```
Comment (adirh3) [inline game_database.json:11]: Each profile needs to have its own object, like this -
```
  {
    "SonarProfileName": "Diablo® IV by Blizzard",
    "ExeName": "Diablo IV"
  },
  {
    "SonarProfileName": "Diablo® IV by Blizzard",
    "ExeName": "Diablo III"
  },
  {
    "SonarProfileName": "Diablo® IV by Blizzard",
    "ExeName": "PathOfExile"
  },
```
The software will try to match from top to bottom

**Review by adirh3 — CHANGES_REQUESTED** [review]

File: game_database.json (hunk 1 above)
Comment (ThePastorJ) [reply game_database.json:11]: Okay, that makes sense. I'll update

**Review by adirh3 — APPROVED** [review]
Looks good, thanks!

**Comment by adirh3** [comment]
> Just updated. I'm making the assumption that any rules made in the app will override the database rules? If not, that'd be a change I'd suggest so people can make their own tweaks on what I've put here.

Yes any custom profiles setup by users will apply over GitHub ones

**Comment by ThePastorJ** [comment]
Just built, works like a charm. Thanks for the feature!

**Pull request merged**